### PR TITLE
Add script to install systemd teabot.service file and start it.

### PR DESCRIPTION
**Pull request by alsuren**
run `./install.sh --fake` to install a service that posts a single teapot per restart to the heroku teabot_endpoints installation at https://teabot-alsuren.herokuapp.com/static/index.html .

run `./install.sh` to set up a real teabot service that reads from the USB scales etc, and posts to the main teabot_endpoints installation.

This is based on my existing input-refactor branch.

### CODE REVIEW

**Comment by alsuren** [comment]
Note that my raspberry pi had problems with pip after the first installation, so I also had to run
```
wget https://bootstrap.pypa.io/get-pip.py
sudo python get-pip.py
```
as described in https://stackoverflow.com/a/27439807

**Comment by AaronKalair** [comment]
Not sure why this diff has the changes from your other PR after I merged it :/

**Comment by alsuren** [comment]
No idea either. I just rebased and force-pushed for your viewing pleasure.

There are actually a few fixups to my previous branch that I probably should have cherry-picked there.

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    exit 1
+fi
+
+if echo $* | grep -q -- --fake
+then
+    ARGS=" --fake"
+fi
+
+cat teabot.service.template \
+    | sed "s:{{PWD}}:$PWD:g" \
```
Comment (AaronKalair) [inline install.sh]: Where does PWD get defined?

**Comment by alsuren** [comment]
I just rebased again to be easier to review (code is identical to the original branch though). The first commit avoids a logging explosion (I left my pi running for an hour and journalctl was full of output from that line).

File: install.sh (hunk 1 above)
Comment (alsuren) [inline install.sh]: $PWD is set to the current directory by the shell, whenever you cd anywhere. I will put it in a temporary variable if you can think of a better name for it.

Comment (AaronKalair) [inline install.sh]: Ah cool didn't know that, that's fine 👍

File: teabot.service.template [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Teabot - tea reporting daemon.
+After=network.target
+
+[Service]
+Type=simple
+User=root
+ExecStart={{PWD}}/run {{ARGS}}
+Restart=always
```
Comment (AaronKalair) [inline teabot.service.template]: Ah so SystemD also does process supervision, that's nice, means we don't need supervised now

**Comment by AaronKalair** [comment]
:whale:

**Pull request merged**